CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Breaking Changes
 
 ### Enhancements
+* Add offset based pagination with beadledom-pagination.
```

Comment:
Since this is a whole new maven module this should probably go under an Additions header.